src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String COMMA = ",";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] data = readFile(fromFileName);
+        int totalSupply = 0;
+        int totalBuy = 0;
+        for (int amount : data) {
+            if (amount > 0) {
+                totalSupply += amount;
+            } else {
+                totalBuy -= amount;
+            }
+        }
+        String report = createReport(totalSupply, totalBuy);
+        writeToFile(report, toFileName);
+        System.out.println("Statistics generated and written to " + toFileName);
```

Comment:
please remove this println(). It seems to be something you've used for testing purposes